.github/workflows/publish-docs.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish Docs
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: The tag to build Docs for
+        required: false
+
+jobs:
+  publish-docs:
+    runs-on: ubuntu-latest
+    if: ${{ inputs.tag != '' }}
+    permissions:
+      pull-requests: write
+    steps:
+    -   name: Checkout sources
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.tag }}
+
+    -   name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '18'
+
+    -   name: Install dependencies
+        run: yarn
+    
+    -   name: Cut a new version
+        run: cd docs && yarn docusaurus docs:version ${{ inputs.tag }}
```

Comment:
This creates changes to the docs that need to be committed, no? Looks like we'd undo the new version in the next deploy.

Reply:
No, it will only deploy a preview, there's a flag down below that would need to be set true for prod